.github/workflows/CreateBuildTag.yaml

```diff
@@ -4,13 +4,13 @@ on:
   workflow_run:
     workflows: ['CI/CD']
     types: ['completed']
-    branches: [ 'main', 'release/*']
+    branches: ['main', 'release/*']
 
 run-name: Create build tag on branch ${{ github.ref_name }}.
 
 jobs:
   TagSuccessfulBuild:
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    if: github.event.workflow_run.conclusion == 'success' && github.repository_owner == 'Microsoft'
```

Comment:
Is yaml case-sensitive? It seems the repo owner is 'microsoft' all lower-case

Reply:
Nice catch ;)